services/movie_session.py

```diff
@@ -0,0 +1,43 @@
+from db.models import MovieSession
+from datetime import datetime, date
+
+
+def create_movie_session(movie_show_time: date,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id)
+
+
+def get_movies_sessions(session_date: str = None) -> MovieSession:

```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]
+
invalid return typehint